qubes_config/global_config.glade

```diff
@@ -492,7 +492,7 @@
                               <object class="GtkImage">
                                 <property name="visible">True</property>
                                 <property name="can-focus">False</property>
-                                <property name="tooltip-text" translatable="yes">This qube will be used as template for disposable qubes launched by all qubes that have "disposable qube template" set to default.</property>
+                                <property name="tooltip-text" translatable="yes">A disposable template is an app qube on which disposables are based. When you start a disposable, it will be based on this app qube by default. &lt;a href="https://www.qubes-os.org/doc/how-to-use-disposables/"&gt;Learn how to use disposables.&lt;/a&gt;</property>
```

Comment:
Not sure if links will work inside of tooltips. If not, let me know.

Reply:
They will not, sadly.

Reply:
No worries. I'll remove the link, then.